contributor/outline.md

```diff
@@ -11,21 +11,22 @@ Outline for this section inspired by the [Trusted Committer introduction video].
 
 ## Becoming an InnerSource Contributor
 
-  - A contributing mindset starts with a shared use mindset.
-    - When you have technical needs that are outside of the domain of your project,
+  - A contributing mindset starts with a sharing mindset.
+    - When you have technical needs that are orthoganal of the domain of your project,
```

Comment:
I'm pretty sure you have changes that are **orthogonal** not **orthoganal**, also they likely are orthogonal **to** not **of**? 
```suggestion
    - When you have technical needs that are orthogonal to the domain of your project,
```